### ISSUE TITLE
Steam некорректно грузит страницы

### ISSUE THREAD

**Issue by Fan4eGGG** [anticensority/runet-censorship-bypass#175]
добрый день.
хотел поинтересоваться, можно ли сделать что то со страницами игр в Стим?

например [вот так](https://prnt.sc/-HGnXSUXbtHe) страница грузится с ВПН и отображается корректно.
а без впн она грузится сломанной и выглядит [вот так.](https://prnt.sc/TZAOlBPlbB_Y).
не знаю, проблема ли это расширения Антизпрета, но вот у меня подобное случается в Опере, хотя в Хром все работает отлично.

благодарю.

**Comment by ilyaigpetrov** [comment]
Нужно найти ломающиеся запросы.
У меня страница team fortress 2 на steam отображается без проблем, так что
я не могу выявить проблемные запросы на своей машине.
Но вы можете попробовать сделать это самостоятельно: см.
https://github.com/anticensority/runet-censorship-bypass/wiki/Чиним-сайты-со-сломанными-элементами
.

**Comment by Fan4eGGG** [comment]
просто это начало ломаться буквально недели 2 назад. и не только страницы с играми (тф2 как пример, они все такие), но и так же не грузятся картинки в гайдах и скриншотах.
хотелось бы решить проблему, но не знаю как.

**Comment by ilyaigpetrov** [comment]
Пришлите снимок экрана, как описано [здесь](https://github.com/anticensority/runet-censorship-bypass/wiki/Чиним-сайты-со-сломанными-элементами#через-инструменты-разработчика-в-браузере-devtools).

**Comment by Fan4eGGG** [comment]
боюсь все не поместится, в нескольких частях кину.

[первая](https://prnt.sc/oQFI4tdMj3V3).
[вторая](https://prnt.sc/1sJJgvhOcOb8).
[третья](https://prnt.sc/C1nhSe5ke0_1).

**Comment by ilyaigpetrov** [comment]
1) Попробуйте открыть store.akamai.steamstatic.com в новой вкладке. Что отобразится? Какая ошибка? Пришлите снимок.
2) Попробуйте открыть любую сломавшуюся картинку (клик правой кнопки -> Открыть изображение в новой вкладке / Open image in new tab). Что отобразится? Какая ошибка? Пришлите снимок.

**Comment by Fan4eGGG** [comment]
если ошибка браузера, то пишет "_Нет подключения к Интернету
На прокси-сервере возникла проблема или адрес указан неверно_."

а если в девтулсе только [вот это](https://prnt.sc/PROY0ZCI0RsI).

**Comment by ilyaigpetrov** [comment]
В devtools нажмите на надпись "(failed)". Должны появиться подробности ошибки (прямо рядом с надписью, в том же столбце).
Если ширины столбца будет мало, то адекватно увеличьте его.
Пришлите снимок или полное наименование ошибки из столбца.

**Comment by Fan4eGGG** [comment]
показывает [вот это](https://prnt.sc/l6u-APIDUdru)

**Comment by Fan4eGGG** [comment]
так в чем может быть дело?

**Comment by ilyaigpetrov** [comment]
`https://store.akamai.steamstatic.com` возвращает `ERR_PROXY_CONNECTION_FAILED`.

И тут нужно понять две вещи:
  1) Почему этот адрес вообще проксируется через наши прокси? Ни akamai, ни steamstatic в реестре не числятся. Вы уверены, что не добавляли данные адреса в проксируемые?
  2) Почему сервер ломается при проксировании данного адреса?


Я попросил администратора АнтиЗапрета прокомментировать данную ошибку.

**Comment by ilyaigpetrov** [comment]
@Fan4eGGG, а без каких-либо средств обхода Steam ломается?
С той же ошибкой `ERR_PROXY_CONNECTION_FAILED`?
Даже если наше расширение отключено через chrome://extensions?

**Comment by Fan4eGGG** [comment]
> * Вы уверены, что не добавляли данные адреса в проксируемые?

100%

> Даже если наше расширение отключено

ага, тоже самое все. правда теперь еще и главная стима так же ломается)

**Comment by ilyaigpetrov** [comment]
> ага, тоже самое все. правда теперь еще и главная стима так же ломается)

 Так, а ошибка `ERR_PROXY_CONNECTION_FAILED` возникает при отключенном расширении? Проверьте через devtools.

**Comment by Fan4eGGG** [comment]
да да, я же это и имел в виду.

**Comment by Fan4eGGG** [comment]
так что там?

**Comment by ilyaigpetrov** [comment]
> -- Так, а ошибка ERR_PROXY_CONNECTION_FAILED возникает при отключенном расширении? Проверьте через devtools.
> -- да да, я же это и имел в виду.

Т.е. ошибка прокси возникает, даже если наше расширение полностью отключено. Это значит, что ошибка не в расширении. Возможно, ошибка закэшировалась -- попробуйте очистить кэш: см. https://github.com/anticensority/runet-censorship-bypass/wiki/Как-очистить-кэш-в-Хроме-через-DevTools%3F. Убедитесь, что у вас выключены какие-либо другие прокси или VPN.

**Comment by Fan4eGGG** [comment]
ошибка на сайте `store.akamai.steamstatic.com` ошибка в коде да, но страница Стим без расширения грузится нормально, я об этом пытаюсь все время сказать.
прокси я еще на старте проблемы отключил, но дело не в нем оказалась.

**Comment by ilyaigpetrov** [comment]
Иконка с синей лентой -> Вкладка "Исключения / Exceptions" -> Ввести store.akamai.steamstatic.com в появившееся поле и нажать "нет" (не проксировать).

Будет так работать?

**Comment by Fan4eGGG** [comment]
хмм. 50\50. когда то грузится норм, а когда по-старому.

**Comment by ilyaigpetrov** [comment]
Возможно, мы нашли не все адреса, проксирование которых ломает сайт.

Следуйте https://github.com/anticensority/runet-censorship-bypass/wiki/Чиним-сайты-со-сломанными-элементами#через-инструменты-разработчика-в-браузере-devtools.
Поймайте момент, когда сайт ломается.
На вкладке Сеть / Network в devtools попробуйте найти ошибки для доменных имён, отличных от store.akamai.steamstatic.com.
Можно прислать снимок экрана (выши предыдущие снимки в этом обсуждении не открываются без VPN -- попробуйте сменить программу, на flameshot какой-нибудь).

**Comment by 25hz** [comment]
Тоже не грузится главная страница магазина,в консоле ссылается красным значком к сайту https://cdn.akamai.steamstatic.com

**Comment by Andrei030x** [comment]
В исключения (через импорт/экспорт, # НЕ ПРОКСИРОВАТЬ) добавляем эти адреса:

shared.cloudflare.steamstatic.com
store.cloudflare.steamstatic.com
clan.cloudflare.steamstatic.com
cdn.cloudflare.steamstatic.com
avatars.cloudflare.steamstatic.com
community.cloudflare.steamstatic.com
store.steampowered.com
steamuserimages-a.akamaihd.net
